config/locales/en.yml

```diff
@@ -1822,9 +1822,7 @@ en:
       leave_team_error: "An error occured."
       leave_flash: "Successfuly left team %{team}."
     user:
-      blank_password_error: "Password can't be blank!"
-      passwords_dont_match: "Passwords don't match"
-      password_invalid: "Password is invalid!"
-      avatar_too_big: "Avatar file size must be less than 0.2 MB"
```

Comment:
check if the test case https://github.com/Ducz0r/scinote-web/blob/998519066c35e6c19e9bd0d3766ca8eeb986f75d/features/settings_page/profile.feature#L27 still work

Reply:
Good catch! Totally forgot about that...